ods_ci/tests/Resources/Files/triton/triton_tensorflow_gRPC_servingruntime.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: serving.kserve.io/v1alpha1
```

Comment:
We are adding this file because to test it on local without those files we cannot test them , once the all PR's are approved we remove the all the duplicate files